page/partials/team.ejs

```diff
@@ -0,0 +1,35 @@
+<div class="team section">
+    <div class="row">
+        <div class="col-md-5">
+            <div>
+                <div>
+                    <h3>Team</h3>
+                    <p>Team of entrepereneurs and experienced professionals.</p>
+                </div> 
+            </div> <!-- end carousel -->
+                
+        </div>
+        <div class="col-md-7">
+
+            <div class="row">
+                <% for (var i = 0; i < 10; i++) { %>
+                <div class="col-md-6 person-info-container <%if (i > 3 ) {%>hidden-picture is-hidden<%}%>" >
```

Comment:
I am not sure but I think that more performant/normal solution would be to put all hidden elements in the separate container and mark it hidden once.